### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by GILL71** [sunlubo/SwiftFFmpeg#48]
<img width="1554" alt="image" src="https://user-images.githubusercontent.com/20766997/105057132-f86d6080-5a85-11eb-80b5-73993f9936c3.png">

I have trouble with linking dependency to my project. I already install ffmpeg from brew (on the screen) and have 100 errors because Xcode can't find some auto-linked libraries

**Comment by sunlubo** [comment]
Yes, FFmpeg can implemet these features. For iOS, you can reference this [example](https://github.com/sunlubo/SwiftFFmpegDemo-iOS).

**Comment by GILL71** [comment]
But in this example you import SwiftFFmpeg not through spm. How can I fix installation process?

**Comment by sunlubo** [comment]
You should use xcframework like [this](https://github.com/sunlubo/SwiftFFmpeg/pull/41).
